assets/js/modules/analytics-4/components/dashboard/EnhancedMeasurementActivationBanner/SetupBanner.js

```diff
@@ -100,6 +108,12 @@ export default function SetupBanner( {
 		onSubmitSuccess();
 	}, [ setValues, submitChanges, onSubmitSuccess ] );
 
+	useMount( () => {
+		if ( usingProxy && ! isDismissed ) {
+			setShouldTriggerSurvey( true );
+		}
+	}, [ isDismissed, setShouldTriggerSurvey, usingProxy ] );
+
```

Comment:
This hook isn't needed, all we need is to render `SurveyViewTrigger` when `isDismissed === false`.

Reply:
Oh gosh, of course, sorry, had a bit of tunnel vision on the change 😆